.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+.env
```

Comment:
You're missing a new line here:
https://stackoverflow.com/a/729795/1056961

Reply:
There's a lot more missing from this .gitignore file - i would Google "node gitignore" and copy the example from Github's own example.

Reply:
I think I've completed this, will send another pull request